### PR TITLE
Add title and description in schemas.json

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Implementation of #34

### CODE REVIEW

File: web/collections/_documentation/integration-autres-systemes.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ Un extrait du fichier `schemas.json` (accessible directement à l'adresse <https
     "schemas":[
         {
             "name":"etalab/schema-decp-dpa",
+            "title": "Spécification du fichier de déclaration de profil d'acheteur",
+            "description": "Spécification du fichier de déclaration de profil d'acheteur"
```
Comment (abulte) [inline web/collections/_documentation/integration-autres-systemes.md]: missing `,`

**Comment by abulte** [comment]
@AntoineAugusti can't see any `description` here 🤔 https://deploy-preview-35--schema-datagouv.netlify.com/schemas/schemas.json

**Comment by AntoineAugusti** [comment]
It doesn't deploy new data content on a branch.

**Comment by abulte** [comment]
Let’s merge then ;-)

**Pull request merged**